nixos/modules/virtualisation/podman/default.nix

```
@@ -5,7 +5,7 @@ let
 
   inherit (lib) mkOption types;
 
-  podmanPackage = (pkgs.podman.override {
+  podmanPackage = ((cfg.package or pkgs.podman).override {
```

`cfg.package` is already Podman by default, or was this required to get it to eval?

```suggestion
  podmanPackage = (cfg.packageoverride {
```

---

I'm not really sure where `cfg.packageoverride` comes from. Is this generated somehow?

---

Typo, sorry. Edited, does it make sense now?

---

Makes total sense! Thanks for the suggestion